Objects/unicodeobject.c

```diff
@@ -15228,13 +15228,16 @@ PyDoc_STRVAR(unicode_doc,
 "str(object='') -> str\n\
 str(bytes_or_buffer[, encoding[, errors]]) -> str\n\
 \n\
-Create a new string object from the given object. If encoding or\n\
-errors is specified, then the object must expose a data buffer\n\
-that will be decoded using the given encoding and error handler.\n\
-Otherwise, returns the result of object.__str__() (if defined)\n\
-or repr(object).\n\
-encoding defaults to sys.getdefaultencoding().\n\
-errors defaults to 'strict'.");
+Create a new string object from the given object.\n\
+\n\
+If a single argument is given, returns the result of object.__str__()\n\
```

Comment:
```suggestion
If the object argument is given, returns the result of object.__str__()\n\
```
You can call `str(encoding='spam')`. A single argument is provided, but no `__str__()` is called.

The object argument will be made positional only in 3.9, so this wording will likely changed again.